snapcraft/_help.py

```diff
@@ -46,7 +46,7 @@ def _topic_help(module_name, devel):
 def _module_help(module_name, devel):
     try:
         module = importlib.import_module(
-            'snapcraft.plugins.{}'.format(module_name))
+            'snapcraft.plugins.{}'.format(module_name.replace("-", "_")))
```

Comment:
please use single quotes instead of double quotes:

    '-', '_'

It's a small detail, but consistency in the details is what makes python pretty :)